src/app/components/Transcript/index.tsx

```diff
@@ -37,14 +37,18 @@ const Transcript = ({
     return null;
   }
 
+  const formattedTitle = title ? `, ${title}` : '';
+
   return (
     <details css={styles.details}>
       <summary css={styles.summary}>
         <ArrowSvg />
-        <Text size="pica" fontVariant="sansBold" css={styles.summaryTitle}>
-          Read transcripts
-          {title && <VisuallyHiddenText>, {title}</VisuallyHiddenText>}
-        </Text>
+        <span role="text">
+          <Text size="pica" fontVariant="sansBold" css={styles.summaryTitle}>
+            Read transcripts
```

Comment:
Could we lose the 's' on the end of 'transcripts' so its 'transcript' please?

Reply:
Oh yh - also meant to say I will make a follow up PR with Translations